src/components/JobActionsPopover.vue

```diff
@@ -29,8 +33,9 @@ import {
   popoverController
 } from "@ionic/vue";
 import { defineComponent } from "vue";
-import { copyOutline, pinOutline, timeOutline  } from 'ionicons/icons'
+import { copyOutline, flashOutline, pinOutline, timeOutline  } from 'ionicons/icons'
 import { mapGetters, useStore } from 'vuex'
+import { alertController } from '@ionic/vue';
```

Comment:
Import ionic components in a single statement.

Reply:
Done sir.